aws/resource_aws_codepipeline_test.go

```diff
@@ -488,10 +592,29 @@ func testAccPreCheckAWSCodePipeline(t *testing.T) {
 	}
 }
 
+func testAccPreCheckAWSCodePipelineAlternateRegion(t *testing.T) {
```

Comment:
Ah its a bummer we cannot reuse logic similar to `testAccPreCheckAWSCodePipeline` here (using a real API call to automatically enable/skip the test), but I see the problem is that we don't initialize an "alternate" global instance of the provider in the testing similar to `testAccProvider`. I think this is okay as-is for now until we can figure out things as part of https://github.com/terraform-providers/terraform-provider-aws/issues/8983 / https://github.com/terraform-providers/terraform-provider-aws/issues/8316

Reply:
Yeah, I briefly considered the option of rolling my own connection to test the API, but I figured the hard-coded option was quicker. Though it actually might not be that complicated